apps/docs/docs/how-to-guides/troubleshooting/general-guideliness.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 0
+description: General guideliness on how to debug a FastStore.
+tags: 
+    - performance
+    - troubleshooting
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Troubleshooting FastStore
+
+An e-commerce is a very complex website were multiple scripts and APIs interact. Debugging such system can be challenging, however, we believe there are general guideliness you can follow to pinpoint the exact origin of the error and than work on a fix. 
```

Comment:
We're using "ecommerce" throughout the docs so we maybe should stick with that form.

Reply:
Hahaha, yea! Slackbot would remind us of that 😅